### PR TITLE
Fix for Galleon prosvisioning should ignore MAVEN_ARGS_APPEND meant f…

### PR DESCRIPTION
**Pull request by jfdenise**
…or S2I app build
ISSUE: https://issues.redhat.com/browse/CLOUD-3847

### CODE REVIEW

**Review by luck3y — APPROVED** [review]

File: jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/galleon/s2i_galleon [hunk 1]
```diff
@@ -158,6 +162,10 @@ function galleon_reset_mvn_env() {
     if [ -n "$ORIGINAL_MAVEN_ARGS" ]; then
       MAVEN_ARGS=$ORIGINAL_MAVEN_ARGS
```
Comment (luck3y) [inline jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/galleon/s2i_galleon:163]: I think this is safer written as:

MAVEN_ARGS=${ORIGINAL_MAVEN_ARGS}

as the args are quite likely to have spaces etc.

File: jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/galleon/s2i_galleon [hunk 2]
```diff
@@ -158,6 +162,10 @@ function galleon_reset_mvn_env() {
...
     fi
+
+    if [ -n "$ORIGINAL_MAVEN_ARGS_APPEND" ]; then
+      MAVEN_ARGS_APPEND=$ORIGINAL_MAVEN_ARGS_APPEND
```
Comment (luck3y) [inline jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/galleon/s2i_galleon:167]: same comment as above

File: jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/galleon/s2i_galleon (hunk 1 above)
Comment (jfdenise) [reply jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/galleon/s2i_galleon]: Thank-you, fixed for both MAVEN_ARGS and MAVEN_ARGS_APPEND

File: jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/galleon/s2i_galleon (hunk 2 above)
Comment (jfdenise) [reply jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/galleon/s2i_galleon]: Thank-you, fixed for both MAVEN_ARGS and MAVEN_ARGS_APPEND

**Review by luck3y — APPROVED** [review]

**Pull request merged**